### DISCUSSION TITLE
android, how to include other libraries?

### DISCUSSION THREAD

**Discussion by Gibbz** [slint-ui/slint#7220] [Q&A]
So I'm paying with this again to see if it's possible to make a simple android app. The last issue im stuck at is using other libraries alongside slint.

Normally they're included in the gradle build files. But slint doesn't seem to have any I can edit.

Has anyone managed to get jni-utils or another library working with slint?

**Comment by ogoffart** [comment]
What kind of other library do you want to use? Java libraries?

**Comment by Gibbz** [comment]
Yes aar libraries which include both java and rust.
I believe I may have just got it working today. I need to do a bit more testing

**Comment by Gibbz** [comment]
Looks like it's working. So I'll close this 👍
